app/templates/treeCounter.html

```diff
@@ -5,44 +5,37 @@
     <title>Trees Planted by Ecosia Users</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <style>
+        @import url('http://fonts.cdnfonts.com/css/campton');
+        body {
+            font-family: 'Campton', sans-serif;
+        }
         img {
             width: 200px;
             height: auto;
         }
         .page-contents {
-            display: flex;
+            display: block;
             justify-content: center;
             margin: auto;
             margin-top: 80px;
+            padding: 40px 40px 60px 40px;
```

Comment:
```suggestion
            padding: 40px 40px 60px;
```

Reply:
In CSS, if the padding option is given 3 values, then the 2nd is applied to right *and* left :)

Reply:
I ended up removing this in my recent changes but still good info to know imho